src/components/curator-actions/fetch-signatories.ts

```diff
@@ -1,11 +1,27 @@
 import { get } from 'svelte/store';
-import { dotApi } from '../../stores';
+import { dotApi, proxies } from '../../stores';
+import type { MultisigInfo } from '../../types/account';
+import { currentBlockchain } from '../app-bar/blockchains';
+
+export async function fetchAllProxies() {
+	if (get(proxies) !== undefined) {
+		return;
+	}
```

Comment:
When changing to Paseo we need to re-fetch

Reply:
true [732a285](https://github.com/galaniprojects/Polkadot-Bounty-Manager/pull/491/commits/732a2850f3ac8217422091fdeaef382b990d8aa8)